webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPNodeGroupRepository.scala

```diff
@@ -748,6 +749,17 @@ class WoLDAPNodeGroupRepository(
     }
   }
 
+  override def createPolicyServerTarget(target : PolicyServerTarget, modId: ModificationId, actor:EventActor, reason:Option[String]): IOResult[LDIFChangeRecord] = {
+    for {
+      con    <- ldap
+      categoryEntry    <-  getCategoryEntry(con, ???).notOptional(s"Entry with ID '${id}' was not found")
```

Comment:
Here I don't know how to get the parentDN needed in `ruleTargetModel`

Reply:
on which branch of LDAP directory does it go?